spec.bs

```diff
@@ -8254,6 +8532,32 @@ A <dfn>trusted scoring signals reply</dfn> is a [=struct=] with the following [=
   :: An {{unsigned long}} or null.
 </dl>
 
+A <dfn>scoring compression group</dfn> is a collection of [=scoring partitions=] that can be compressed
+together in the signals response. It's a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="scoring compression group">
+  : <dfn>compression group id</dfn>
+  :: An integer indicates the index of this compression group.
+  : <dfn>partitions</dfn>
+  :: A [=list=] of [=scoring partition=]. Contains all the [=scoring partitions=] belong to this compression
+    group.
+</dl>
+
+A <dfn>scoring partition</dfn> is a collection of [=trusted scoring signals request/render URLs=] and 
+[=trusted scoring signals request/ad component URLs=] that can be processed together by the service
+without any potential privacy leakage. It's a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="scoring partition">
+  : <dfn>id</dfn>
+  :: An integer indicates the index of this partition.
+  : <dfn>namespace</dfn>
+  :: A [=map=], whose [=map/keys=] are [=strings=] and [=map/values=] are [=list=] of

```

Comment:
it seems this `scoring partition` is the same as `bidding partition`, but just the namespace's keys are different? If that's the case, you can probably just use the same struct called `partition`, and explain that for bidding, it contains ...., and for scoring, it contains .... Then define your `scoring parition` as "a <dfn>scoring parition</dfn> is a [=partition=], for a collection of ....", and same for `bidding partition`.

Reply:
but I'm fine to change it in your follow up PR, since it's more of readability change.

Reply:
Gotcha, will change this in the follow up PR.